### ISSUE TITLE
wp8 with scan method but trigger "nothing"

### ISSUE THREAD

**Issue by bau720123** [phonegap/phonegap-plugin-barcodescanner#17]
hi all
for now I am using this version in PGB 3.7
```
<gap:plugin name="phonegap-plugin-barcodescanner" source="npm" version="3.1.2" />
```
when I use scan method in wp8，it seem can scan the QR CODE perfectly，but after scan，nothing happened...
no trigger on alert，just "nothing"

**Comment by ZoRDoK** [comment]
Similar issue on android 4.4.2. Events not triggered after scan qr, but fail on timeout with cancelled.

**Comment by bau720123** [comment]
problem solved in 4.0.0 version
thanks
